doc/PLUGIN.md

```diff
@@ -0,0 +1,44 @@
+# Plugin API
+
+`Miradord` makes use of external plugins to define and enforce spend
+policies.
+
+## Hooks
+
+### `new_block`
```

Comment:
I think we should rename it to `new_tip`. I think `new_block` is confusing as it gives the wrong impression that we would somehow send them all the new blocks.

Reply:
Or maybe even something else.

Reply:
we will chang it in another PR https://github.com/revault/miradord/blob/580a0a305c2bd9f3fb0fd2a90893d363e29b34f0/src/plugins.rs#L129 with the code change